ospfd/ospf_vty.c

```diff
@@ -2229,6 +2229,24 @@ ALIAS(no_ospf_compatible_rfc1583, no_ospf_rfc1583_flag_cmd,
       "OSPF specific commands\n"
       "Disable the RFC1583Compatibility flag\n")
 
+DEFPY (ospf_send_extra_data,
+       ospf_send_extra_data_cmd,
+       "[no] ospf send-extra-data zebra",
+       NO_STR
+       OSPF_STR
+       "Extra data to Zebra for display/use\n"
+       "To zebra\n")
+{
+	VTY_DECLVAR_INSTANCE_CONTEXT(ospf, ospf);
+
+        if (no)
+		UNSET_FLAG(ospf->config, OSPF_SEND_EXTRA_DATA_TO_ZEBRA);
```

Comment:
what purpose does this CLI slove? We could always send this extra data by default to zebra and we can control with CLI to really show it not? Thoughts?

Reply:
The idea is to allow operators to disable opaque attributes to consume less memory if they aren't needed (e.g. low-end routers with memory limitations). The same command exists in _bgpd_, from where it was copied from.